msu/systems/keybinds/keybind.nut

```diff
@@ -81,8 +68,13 @@ this.MSU.Class.Keybind <- class
 		return setting;
 	}
 
+	function tostring()
```

Comment:
Should probably be `toString` and `_toString`, no?

Reply:
These are squirrel names for these functions, the same as what is used for integers/booleans. and `_tostring` is a metamethod and has to be named that way.